content/episodes/2019-08-12-S05E05-the-5g-privilege.md

```diff
@@ -0,0 +1,165 @@
+---
+date: 2019-08-12T04:55:00-08:00
+title: "The 5G Privilege"
+ogtitle: "The 5G Privilege"
+description: |
+    ‘5G’ is a new buzzword floating around every corner of the internet. But what exactly is this hyped-up cellular network, often referred to as the next technological evolution in mobile internet communications? Will it really be 100 times faster than what we have now? What will it make possible that has never been possible before? Who will reap the benefits? And, who will get left behind?
+
+    Mike Thelander at [Signals Research Group](https://signalsresearch.com/) imagines the wild ways 5G might change our lives in the near future. [Rhiannon Williams](https://twitter.com/RhiannonJudithW/) hits the street and take a new 5G network out for a test drive. [Amy France](https://twitter.com/amy_kaymay/) lives in a very rural part of Kansas — she dreams of the day that true, fast internet could come to her farm (but isn’t holding her breath). [Larry Irving](https://twitter.com/larry_irving/) explains why technology has never been provided equally to everyone, and why he fears 5G will leave too many people out. [Shireen Santosham](https://twitter.com/ssantosham/), though, is doing what she can to leverage 5G deployment in order to bridge the digital divide in her city of San Jose.
```

Comment:
this sentence needs a small rewrite:

> Rhiannon Williams hits the street and take a new 5G network out for a test drive.

Reply:
Changed to "Rhiannon Williams hits the street to test drive a new 5G network."